extensions/dou-tu/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# 斗图 Changelog
+
+## [Add Dou Tu Extension] - 2022-08-31
+- 支持快速搜索表情包
+- 支持点击复制到剪切板
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.